modin/engines/ray/task_wrapper.py

```diff
@@ -11,19 +11,72 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+The module with helper mixin for executing functions remotely.
+
+To be used as a piece of building a Ray-based engine.
+"""
+
 import ray
 
 
 @ray.remote
 def deploy_ray_func(func, args):  # pragma: no cover
+    """
+    Wrap `func` to ease calling it remotely.
+
+    Parameters
+    ----------
+    func : callable
+        A local function that we want to call remotely.
+    args : dict
+        Keyword arguments to pass to `func` when calling remotely.
+
+    Returns
+    -------
+    ray.ObjectRef or list
+        Ray identifier of the result being put to Plasma store.
```

Comment:
In different places we use "plasma"/"Plasma". Which one is preferable?

Reply:
Well, Ray in their docs uses `Plasma` spelling, so we should probably use that, too.